app/components/govuk_component/summary_list_component/action_component.rb

```diff
@@ -10,8 +10,9 @@ def initialize(href: nil, text: 'Change', visually_hidden_text: false, classes:
 
     super(classes: classes, html_attributes: html_attributes)
 
-    @href = href
-    @text = text
+    @href       = href
+    @text       = text
+    @action_suffix = action_suffix
```

Comment:
Will fix the wonky alignment 🤦🏽‍♂️

Reply:
The alignment was because I did a find and replace for `card_title` to `action_suffix` 😅